planning/motion_velocity_planner/autoware_motion_velocity_run_out_module/src/debug.hpp

```diff
@@ -0,0 +1,365 @@
+// Copyright 2025 TIER IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DEBUG_HPP_
+#define DEBUG_HPP_
+
+#include "types.hpp"
+
+#include <autoware/motion_utils/trajectory/interpolation.hpp>
+#include <autoware/motion_utils/trajectory/trajectory.hpp>
+#include <autoware/motion_velocity_planner_common/velocity_planning_result.hpp>
```

Comment:
We should change like this. The same should be done in other inclusions.

```suggestion
#include <autoware/motion_velocity_planner_common_universe/velocity_planning_result.hpp>
```

Reply:
Thank you. I just pushed the change